docs/deploying.md

```diff
@@ -69,119 +69,110 @@ Each service's `main` package is located in the `cmd` directory.
->       -var repo_owner=${YOUR_REPO_OWNER} \
->       -var repo_name=${YOUR_REPO_NAME} \
->       -var cloudsql_tier="db-custom-1-3840" \
->       -var cloudsql_disk_size="16"
-> ```
+1. Authenticate with Google Cloud using the `gcloud` command-line tool:
 
+   ```console
+   gcloud auth login && gcloud auth application-default login
+   ```
 
-electin vCPU and Postgres size (concurrent connections):
+   This will open two authentication windows in your web browser.
 
-PostgresSql sizing
-And pricing
-https://cloud.google.com/sql/docs/postgres/create-instance
-Combined with connection limits: https://cloud.google.com/sql/docs/quotas#cloud-sql-for-postgresql-connection-limits
-Choice for now:
+   You may need to `unset GOOGLE_APPLICATION_CREDENTIALS` as it takes precedence
+   over gcloud's login settings.
 
-db-custom-8-30720
+1. Change to this directory and run `terraform init`.  Terraform will
+   automatically download the plugins required to execute this code.
 
+1. Run `terraform apply` to start deployment:
 
-30 gb which gives 500 concurrent connections
+   Without Cloud Build Triggers:
 
+   ```console
+   terraform apply \
+     -var project=$PROJECT_ID
+   ```
 
+   With Cloud Build Triggers:
 
+   ```console
+   terraform apply \
+     -var project=${PROJECT_ID} \
+     -var region="us-central-1" \
+     -var use_build_triggers=true \
+     -var repo_owner=${YOUR_REPO_OWNER} \
+     -var repo_name=${YOUR_REPO_NAME}
+   ```
 
-1.  Initialize and/or Migrate the DB.
-
-    > **NOTE** In the future this may be handled by terraform
-    
-    To migrate the database, you will want to start the
-    [Cloud SQL Proxy](https://cloud.google.com/sql/docs/postgres/quickstart-proxy-test#install-proxy)
-    and then run the [migrate](https://github.com/golang-migrate/migrate)
-    command.
-    
-    ```text
-    $ DB_HOST="localhost"
-    $ DB_PORT="1433"
-    $ DB_USER="notification"
-    $ DB_PASSWORD="YOUR-DB-PASSWORD"
-    $ DB_SSLMODE="disable"
-    $ DB_NAME="main"
-    $ DB_URL="postgres://${DB_USER}:${DB_PASSWORD}@${DB_HOST}:${DB_PORT}/${DB_NAME}?sslmode=${DB_SSLMODE}"
-
-    $ migrate -database ${DB_URL} -path ./migrations up
-    ```
-
+Terraform will begin by creating the service accounts and enabling the services
+on Google Cloud that are required to run this server.
+
+#### Local development and testing example deployment
+
+The default Terraform deployment is a production-ready, high traffic
+deployment. For local development and testing, we recommend you use the
+following sample deployment:
+
+1. Run `terraform apply` with the following command:
+
+   ```console
+   terraform apply \
+     -var project=${PROJECT_ID} \
+     -var region="us-central-1" \
+     -var use_build_triggers=true \
+     -var repo_owner=${YOUR_REPO_OWNER} \
+     -var repo_name=${YOUR_REPO_NAME} \
+     -var cloudsql_tier="db-custom-1-3840" \
+     -var cloudsql_disk_size_gb="16"
+   ```
+
+1. Initialize or migrate the database.
```

Comment:
This is part of the broader steps. Is this getting grouped under the 'testing deployment'?

Reply:
Sorry misunderstood previous message, moved this into the main section, leaving the local dev and test section for only a scaled down deployment.